openid-sharedsignals-framework-1_0.md

```diff
@@ -787,9 +787,7 @@ following properties:
 
 stream_id
 
-> **Transmitter-Supplied**, A string that uniquely identifies the stream. Stream
-  IDs MUST be unique per Reciever.  This value is generated by the Transmitter
-  when the stream is created.
+> **Transmitter-Supplied**, A string that uniquely identifies the stream. A transmitter MUST generate a unique ID for each of it's streams at the time of stream creation.
```

Comment:
it's streams -> its open streams (no apostrophe, and specify open, since the rest of the sentence talks about the time of creation)